App/UI/Onboarding/Shared/OnboardingHeaderCell.swift

```diff
@@ -22,17 +22,22 @@ import Tempura
 struct OnboardingHeaderCellVM: ViewModel, Equatable {
   let title: String
   let description: String
+  let actionButtonTitle: String
```

Comment:
[i] This has always the same value: `L10n.Onboarding.Common.discoverMore`.
Not sure I would have passed it in the VM. That said, it's more generic and it's fine as is.